modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/util/IgniteMethod.java

```diff
@@ -98,7 +98,10 @@ public enum IgniteMethod {
     IS_NOT_DISTINCT_FROM(Objects.class, "equals", Object.class, Object.class),
 
     /** See {@link IgniteSqlFunctions#skipFirstArgument(Object, Object)}. **/
-    SKIP_FIRST_ARGUMENT(IgniteSqlFunctions.class, "skipFirstArgument", Object.class, Object.class);
+    SKIP_FIRST_ARGUMENT(IgniteSqlFunctions.class, "skipFirstArgument", Object.class, Object.class),
```

Comment:
This is not a last method in this class now. GitHub should show conflict here, but for some reason doesn't show it. Please merge master to branch.